assets/webpack.config.js

```diff
@@ -4,6 +4,7 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+const webpack = require('webpack');
```

Comment:
What are these changes for? They don't seem to be doing anything?

Reply:
Oops that was from a SO post I tried while trying to get D3 to be visible. Deleted the other code but forgot that line. Should be safe to remove